storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -1,51 +1,51 @@
 :_content-type: ASSEMBLY
 [id="persistent-storage-csi-ebs"]
-= AWS Elastic Block Store CSI Driver Operator
+= Amazon Elastic Block Store CSI Driver Operator
 include::_attributes/common-attributes.adoc[]
 :context: persistent-storage-csi-ebs
 
 toc::[]
 
 == Overview
 
-{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic Block Store (EBS).
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Amazon Elastic Block Store (EBS).
 
 Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
 
-To create CSI-provisioned PVs that mount to AWS EBS storage assets, {product-title} installs the AWS EBS CSI Driver Operator and the AWS EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+To create CSI-provisioned PVs that mount to Amazon EBS storage assets, {product-title} installs the Amazon EBS CSI Driver Operator and the Amazon EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
 
-* The _AWS EBS CSI Driver Operator_ provides a StorageClass by default that you can use to create PVCs. You can disable this default storage class if desired (see xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#persistent-storage-csi-sc-manage[Managing the default storage class]). You also have the option to create the AWS EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using AWS Elastic Block Store]. 
+* The _Amazon EBS CSI Driver Operator_ provides a StorageClass by default that you can use to create PVCs. You can disable this default storage class if desired (see xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#persistent-storage-csi-sc-manage[Managing the default storage class]). You also have the option to create the Amazon EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using Amazon Elastic Block Store]. 
 
-* The _AWS EBS CSI driver_ enables you to create and mount AWS EBS PVs.
+* The _Amazon EBS CSI driver_ enables you to create and mount Amazon EBS PVs.
```

Comment:
Revoked, Amazon EBS CSI driver -> AWS EBS CSI driver.

Reply:
@smunje1 It doesn't seem like this title is linking out to OCP resources or code that reference "AWS EBS". "Amazon EFS CSI driver" is the appropriate name to be used for the driver. Is there a reason I'm missing why this update should be revoked?

https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html

Reply:
Revoked. As the first service name instance is associated with source repo name and the second instance with resource.